test/integration/select_readview_test.lua

```diff
@@ -2321,7 +2321,19 @@ pgroup.test_stop_select = function(g)
     t.assert_equals(err, nil)
 end
 
+pgroup.after_test('test_stop_select', function(g)
+    -- It seems more easy to restart the cluster rather then restore it
+    -- original state.
+    if g.params.backend == helpers.backend.VSHARD then
+        helpers.stop_cluster(g.cluster, g.params.backend)
+        g.cluster = nil
+        init_cluster(g)
+    end
+end)
+
 pgroup.test_select_switch_master = function(g)
```

Comment:
I think here we should support vshard scenario too. But since I have no idea what's the proper switchover for vshard (and maybe it doesn't have it yet), we may just file a `TO DO:` here or/and file a follow up ticket.

Reply:
Added: https://github.com/tarantool/crud/issues/383